.codecov.yml

```diff
@@ -0,0 +1,19 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
+coverage:
+  precision: 1
+  round: nearest
+  range: '10...100'
+
+  status:
+    project:
+      default:
+        target: auto
+        threshhold: 25
+        base: pr
+    patch: no
+    changes: no
+
+comment: off
```

Comment:
*no* commenting by codecov with coverage data on PRs

Reply:
Do you think this is something we'll want to turn on down the track? What does the commenting look like?

Reply:
perhaps, once we tune our thresholds and whatnot! it looks like this: https://docs.codecov.io/docs/pull-request-comments